src/main/java/spark/Service.java

```diff
@@ -130,6 +131,16 @@ public synchronized Service port(int port) {
     }
 
     /**
+     * Set whether to enable the underlying server's JMX counters
+     * @param enableServerJMX
+     * @return
+     */
+    public synchronized Service enableServerJMX(boolean enableServerJMX) {
```

Comment:
Does it make sense to disable it again programatically? I imagine you either want it on or off.
It should be enough to make this disabled by default and give users the option to enable it. 
`enableServerJmx()`(no args).


Is having this running resource intensive? If not it could be enabled by default 🤔 

You probably can't do this after the server has started, so add

```
if (initialized) {
    throwBeforeRouteMappingException();
}
```